.github/workflows/build-size-comparison.yml

```
@@ -0,0 +1,138 @@
+        cd ../../base-branch/build/stackable
+        zip -r ../../../base-build.zip . -x "*.map" "node_modules/*"
+        
+        # Move zip files to root directory for easier access
+        cd ../../../
+        ls -la *.zip
+
+    - name: Compare Build Sizes
+      run: |
+        echo "## 📦 Build Size Comparison" >> $GITHUB_STEP_SUMMARY
+        echo "" >> $GITHUB_STEP_SUMMARY
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)